pkg/scheduler/framework/runtime/framework.go

```diff
@@ -208,6 +212,13 @@ func WithExtenders(extenders []framework.Extender) Option {
 	}
 }
 
+// WithParallelizer sets parallelizer for the scheduling frameworkImpl.
+func WithParallelizer(parallelizer parallelize.Parallelizer) Option {
```

Comment:
Should this be `WithParallelism` instead? That way, callers of this method don't need to know about the Parallelizer object.